src/device.c

```diff
@@ -36,34 +38,42 @@ static void device_file_path(char *out, const char *dir) {
 static int device_ini_handle(void *out, const char *section, const char *name,
                              const char *value) {
   device_info_t *info = out;
+
   if (strcmp(name, "paired") == 0) {
     info->paired = BOOL(value);
   } else if (strcmp(name, "internal") == 0) {
     strncpy(info->internal, value, 255);
   } else if (strcmp(name, "external") == 0) {
     strncpy(info->external, value, 255);
+  } else if (strcmp(name, "last_used_address") == 0) {
+    info->last_used_address = INT(value);
```

Comment:
can you change it to boolean type?

Reply:
No more HA multi-WAN failover streaming? Done.